agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -285,9 +401,10 @@ private static List<ProcessorConfig> getSpanProcessorConfigs(Configuration confi
   private static SdkLogEmitterProviderBuilder configureLogging(
       SdkLogEmitterProviderBuilder builder,
       TelemetryClient telemetryClient,
+      @Nullable QuickPulse quickPulse,
```

Comment:
I checked portal, livemetrics supports requests, dependencies, performanceCounters, exception. I don't find it uses for logs.